src/modules/views/Modal/QualifyFileView.jsx

```diff
@@ -1,37 +1,125 @@
 import React from 'react'
 import { useNavigate, useParams } from 'react-router-dom'
 
-import { hasQueryBeenLoaded, useQuery } from 'cozy-client'
-import { EditDocumentQualification } from 'cozy-scanner'
+import { isQueryLoading, useClient, useQuery } from 'cozy-client'
+import { getQualification } from 'cozy-client/dist/models/document'
+import { themesList } from 'cozy-client/dist/models/document/documentTypeData'
+import { isQualificationNote } from 'cozy-client/dist/models/document/documentTypeDataHelpers'
+import flag from 'cozy-flags'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import IconStack from 'cozy-ui/transpiled/react/IconStack'
+import NestedSelectResponsive from 'cozy-ui/transpiled/react/NestedSelect/NestedSelectResponsive'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
 
 import { LoaderModal } from 'components/LoaderModal'
+import { useScannerI18n } from 'lib/ScannerI18nProvider'
 import { buildFileByIdQuery } from 'modules/queries'
 
-const QualifyFileView = () => {
+const getThemesList = () =>
```

Comment:
I think this should be separated into a `helpers` file

Reply:
Vu ensemble, les helpers étant petits et très spécifique au composant, nous choisissons de les laisser ici